src/control/server/ctl_system.go

```diff
@@ -66,24 +82,26 @@ func (svc *ControlService) getMSMemberAddress() (string, error) {
 //
 // TODO: specify the ranks managed by the harness that should be started.
 func (svc *ControlService) updateMemberStatus(ctx context.Context) error {
-	// exclude members with states that can't be updated with response check
-	statesToExclude := []system.MemberState{
-		system.MemberStateEvicted, system.MemberStateErrored,
-		system.MemberStateUnknown, system.MemberStateStopped,
-		system.MemberStateUnresponsive,
-	}
-	hostAddrs := svc.membership.Hosts(statesToExclude...)
+	hostRanks := svc.membership.HostRanks()
 
-	svc.log.Debugf("updating response status for ranks on hosts: %v", hostAddrs)
+	svc.log.Debugf("updating response status for ranks %v", hostRanks)
 
 	// Update either:
 	// - members unresponsive to ping
 	// - members with stopped processes
 	// TODO: update members with ping errors
 	badRanks := make(map[uint32]system.MemberState)
-	for _, addr := range hostAddrs {
+	for addr, ranks := range hostRanks {
 		hResults, err := svc.harnessClient.Query(ctx, addr)
 		if err != nil {
+			if strings.Contains(err.Error(), "connection refused") {
```

Comment:
I think this should be pulled into a helper function and made a little more resilient, e.g.
```go
func isUnreachableError(err error) bool {
        switch ne := errors.Cause(err).(type) {
        case *net.OpError:
                return ne.Temporary()
        }

        return false
}
```